### ISSUE TITLE
Increase file size limit for Log Reports

### ISSUE THREAD

**Issue by GlitchFlux** [Yellow-Dog-Man/Resonite-Issues#2971]
### Is your feature request related to a problem? Please describe.

I'm unsure if it is still an issue like it was for me before, but when reporting bugs and issues you need to submit logs. Issue i have is my logs before used to constantly be a little over the file limit.

### Describe the solution you'd like

Just increase the size limit a few megabytes, either by a quarter or a half.

### Describe alternatives you've considered

Sharing links to logs elsewhere but thats sharing logs to two places instead of just one.

### Requesters

GlitchFlux

**Comment by GlitchFlux** [comment]
Additional context, i think limit is around 10,000kb, my usual is 14,000-12,000 with the largest at 631,272kb

**Comment by Frooxius** [comment]
Are you talking about the file size limit on GitHub?

I don't think that's something we can increase, since that's limit imposed by GitHub itself: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files

However there are two things you can do:
1) Make a clean replication log, which can be much smaller - this is generally preferred if possible
2) Compress the log file into a ZIP - logs are often extremely compressible, so just zipping them up will make their size a fraction of what it normally is

**Comment by ko-tengu** [comment]
The github limits aren't really in the team's control, but a suggestion to compress larger logs may help. With the compression of even Windows' default zip scheme you should be able to fit about 200mb of log into a 10mb zip file. Windows also has 7z compression built in now, which makes my 600mb entire log folder into 11mb.
![image](https://github.com/user-attachments/assets/a826fa0b-0340-459e-ba2d-05cb6c30dcf7)

**Comment by Frooxius** [comment]
I am a bit confused now. We do not have bug tickets on the Resonite page, only moderation tickets. Bugs are reported here on GitHub.

We could potentially increase the upload limit for tickets, that's something @ProbablePrime would probably know more about. 

Compressing should still work as workaround too.

**Comment by GlitchFlux** [comment]
I notice again, bug report stuff isn't there, its github, but i remember a staff member requesting a log posted on one of the tickets i had made, and i gave up on various tickets where my files would be too big. I don't even remember if its an issue that can be fixed as from an end user view everything surrounding the correct channels for different things is very confusing

**Comment by Frooxius** [comment]
Yes, moderation can ask for logs as evidence for a number of things.

I do not understand why did you close the issue? Like I said above, we can potentially increase upload limit for tickets.

**Comment by shiftyscales** [comment]
Do you have any examples of how large some of the files you've previously tried and failed to upload to the ticket system are, @GlitchFlux?

As had already been indicated though- for logs in particular, they are highly compressible, and you can send one to a zip file in Windows by right clicking on it > Send to > Compressed (zipped) folder which should drastically reduce the size / make it possible to upload.

**Comment by GlitchFlux** [comment]
Zipping and compressing makes sense, but also, yes. Judging by the timestamps from when I had made one report and likly stated I couldn't put a log, these are some of the sizes that may have been the heavy hitters:

10,999kb
27,049kb
10,540kb
14,333kb
106,564kb
19,740kb

Those are from around the week of the issue I had back in November, found to be the Twitch Integration that killed Headlesses. Others from that week fall into the 6-10k which should fit just fine

**Comment by ProbablePrime** [comment]
I increased this periodically, it should be at 10mb right now. I'll increase this to 30 when I get the chance.
